app/graphql/kitsu_schema.rb

```diff
@@ -1,3 +1,5 @@
+require 'custom_errors'
```

Comment:
Generally it’s better to use `require_dependency` so Rails can reload this

Reply:
Didn't know about that, thank you.